etc/deploy/azure/azure-doco-script.sh

```diff
@@ -149,7 +154,9 @@ pachd:
     annotations:
       service.beta.kubernetes.io/azure-load-balancer-resource-group: ${RESOURCE_GROUP}
   image:
-    tag: "2.1.1"
```

Comment:
hmm, are these supposed to be regularly updated? Should this actually be blank, to inherit from whatever the actual current chart is? This makes me wonder how customers actually construct their values.yaml

Reply:
So, the short answer is that it has always been explicit, and no-one has ever changed that.

I _think_ it makes sense to keep it explicit, because it means that the script should always create a proper cluster. The alternative would be to test the script with each new version. That requires some work: I’ll create a story for it.